libs/sdk-ui/src/base/localization/bundles/en-US.json

```diff
@@ -1260,5 +1253,60 @@
         "value": "Remove from selection.",
         "comment": "Text of the tooltip shown above button that removes user or group from unsaved selection of newly added grantees that will gain access right to the dashboard after changes in share dialog are saved.",
         "limit": 0
+    },
+    "sorting.dropdown.header": {
+        "value": "Sort insight",
+        "comment": "",
+        "limit": 0
+    },
+    "sorting.type.alphabetical.asc": {
+        "value": "A to Z",
+        "comment": "",
+        "limit": 0
+    },
+    "sorting.type.alphabetical.desc": {
+        "value": "Z to A",
+        "comment": "",
+        "limit": 0
+    },
+    "sorting.type.numerical.asc": {
+        "value": "Smallest to largest",
+        "comment": "",
+        "limit": 0
+    },
+    "sorting.type.numerical.desc": {
+        "value": "Largest to smallest",
+        "comment": "",
+        "limit": 0
+    },
+    "sorting.sum.of.all.measure._measure": {
+        "value": "Sum of all measures (total)",
+        "comment": "",
+        "limit": 0
+    },
+    "sorting.sum.of.all.measure._metric": {
+        "value": "Sum of all metrics (total)",
+        "comment": "",
+        "limit": 0
+    },
+    "sorting.tooltip.sum._measure": {
```

Comment:
tooltip texts should not be hardcoded in sort components and defined here. It should be provided by PV in sortConfig depending on the context of current buckets. 
translations should go to libs/sdk-ui-ext/src/internal/translations/en-US.json, PV should provide them in sortConfig in `explanation` keys and sort components should render that from there only